package.json

```diff
@@ -1,5 +1,5 @@
 {
   "devDependencies": {
-    "cypress": "^13.6.1"
+    "cypress": "^12.17.2"
```

Comment:
does pinning this fix a some test?

Reply:
No. I pinned it to the same version used in docker-compose.test.yml just to be consistent. I am sure 13.6.1 will work but it is not tested yet.